roles/kubernetes-apps/container_engine_accelerator/nvidia_gpu/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+container_engine_accelerator_enabled: false
+nvidia_driver_version: "384.111"
+nvidia_gpu_tesla_base_url: https://us.download.nvidia.com/tesla/
+nvidia_gpu_gtx_base_url: http://us.download.nvidia.com/XFree86/Linux-x86_64/
+nvidia_gpu_flavor: tesla
+nvidia_url_end: "{{nvidia_driver_version}}/NVIDIA-Linux-x86_64-{{nvidia_driver_version}}.run"
+## this should end up in var/ubuntu.yml
+#nvidia_driver_install_container: atzedevries/nvidia-ubuntu-driver-installer:21-1
+nvidia_driver_install_container: gcr.io/google-containers/ubuntu-nvidia-driver-installer@sha256:eea7309dc4fa4a5c9d716157e74b90826e0a853aa26c7219db4710ddcd1ad8bc
+nvidia_driver_installer_supported_distrubion_release:
+  - xenial
+## Download URL of Nvidia GPU drivers.
+## Use this for Tesla based cards
+# nvidia_driver_download_url_default: https://us.download.nvidia.com/tesla/{{nvidia_driver_version}}/NVIDIA-Linux-x86_64-{{nvidia_driver_version}}.run
```

Comment:
The default is not something that the user should be setting. I think the default should be maintained by kubespray and the user can override the `nvidia_driver_download_url`. Or is this leftover and should be deleted?

Reply:
left over and should be deleted. Dowload url is now controlled by the variable `nvidia_gpu_flavor`